### ISSUE TITLE
10$: Web demo has phantom errors in Chromium 95 (Ubuntu)

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#513]
Chromium 94: no repro
Chromium 95 (Snap): repro 
Chromium 95 (Debian): repro

![Screenshot from 2021-11-15 18-19-14 cropped](https://user-images.githubusercontent.com/48666/141883956-d575ea19-ed11-45f6-a16e-a124559b7749.png)

**Comment by strager** [comment]
The bug still happens with Version 96.0.4664.45 (Official Build) for Linux Mint (64-bit), but the symptoms are slightly different.

**Comment by strager** [comment]
Seems like a bug in Chromium. The following is a minimal repro:
```
<!DOCTYPE html>
<html>
  <head>
    <style>
      #code-input,
      #shadow-code-input {
        font-size: 1rem;
      }

      pre mark {
        text-decoration: underline;
        text-decoration-color: red;
        text-decoration-line: underline;
        text-decoration-skip-ink: none;
        text-decoration-style: wavy;
      }

      /* Show #shadow-code-input visually behind #code-input. */
      #code-input {
        background: transparent;
        position: relative;
      }
      #shadow-code-input {
        position: absolute;
      }
    </style>
  </head>
  <body>
    <main>
      <pre id="shadow-code-input"><mark>i</mark></pre>
      <textarea id="code-input" spellcheck="false"></textarea>

      <script>
        setTimeout(() => {
          let shadowCodeInputElement =
            document.getElementById("shadow-code-input");
          shadowCodeInputElement.textContent = "";
          let codeInputElement = document.getElementById("code-input");
          codeInputElement.focus();
        }, 100);
      </script>
    </main>
  </body>
</html>
```

**Comment by strager** [comment]
Worked around in commit ecbd29575c167fb7672e4610d88e906915cd8f7f. I need to file a bug with Chrome upstream.

**Comment by strager** [comment]
> Worked around in commit [ecbd295](https://github.com/quick-lint/quick-lint-js/commit/ecbd29575c167fb7672e4610d88e906915cd8f7f). I need to file a bug with Chrome upstream.

Filed: https://bugs.chromium.org/p/chromium/issues/detail?id=1273133

**Comment by strager** [comment]
The bug seems to be fixed in Chromium Version 97.0.4692.71 (Official Build) for Linux Mint (64-bit).

However, I'm going to keep the workaround in place. Checking for the specific Chromium version in CSS (in order to disable our workaround) sounds like a headache.
